Code/GraphMol/ChemReactions/ReactionUtils.h

```diff
@@ -63,6 +63,8 @@ bool isReactionTemplateMoleculeAgent(const ROMol &mol, double agentThreshold);
 
 void updateProductsStereochem(ChemicalReaction *rxn);
 
+void removeMappingNumbersFromReactions(const ChemicalReaction &rxn);
```

Comment:
SO you know, I might add an update to this function

```
removeMappingNumbersFromReactions(rxn, removeUnusedOnly=false);
````

for my sanitization step in the future.